client/infinispan-key-value-store-hotrod/src/test/java/org/infinispan/api/client/impl/Address.java

```diff
@@ -0,0 +1,70 @@
+package org.infinispan.api.client.impl;
+
+import java.util.Objects;
+
+import org.infinispan.protostream.annotations.ProtoDoc;
+import org.infinispan.protostream.annotations.ProtoField;
+
+@ProtoDoc("@Indexed")
+public final class Address {
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 1, required = true)
+   String number;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 2, required = true)
+   String street;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 3, required = true)
+   String postalCode;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 4, required = true)
+   String town;
+
+   @ProtoDoc("@Field")
+   @ProtoField(number = 5, required = true)
+   String country;
```

Comment:
In general we're trying to avoid setting `required = true` on proto messages as it allows greater flexibility and has actually been removed in `proto3`.

Reply:
ok, I will change all that when I truly understand how it works and when I will be using proto3. 
For now this is only used on my tests